### DISCUSSION TITLE
The fifa+ videos, download without audio and black screen

### DISCUSSION THREAD

**Discussion by delvisomanda** [aclap-dev/video-downloadhelper#785] [Help and Q&A]
### Your question

As the title says, the videos downloaded from the plus.fifa.com page have no audio and black screen. I tried to play them with different players, and even format them to another format, but no luck.

### Addon & CoApp version and details

version: 9.0.2.3
target: google
channel: stable
lang: es-ES
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36
{
    "used_sidebar_button": true,
    "download_directory": "D:\\Archivos Personales\\Downloads",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 2,
    "successfull_dl": 1
}

**Comment by Mattmjs** [comment]
Unfortunately the fifa site uses DRM to protect the content, they wouldn't allow a video to play unless DRM was enabled.
Nothing can be done about it.
